metadata_mapper/map_registry_collections.py

```diff
@@ -8,15 +8,26 @@
 
 def map_endpoint(url):
     lookup = {
-        'ucldc_nuxeo': 'nuxeo'
+        'ucldc_nuxeo': 'nuxeo',
+        'oac_dc': 'oac',
+        'islandora_oai_dc': 'islandora'
```

Comment:
this sort of translation should really happen on the registry side of things, and at some point when we have a firmer "mapper vocabulary" (probably after Rikolti's launch) we should migrate the registry to use the new vocabulary.

Reply:
makes sense